packages/next/build/webpack/plugins/flight-manifest-plugin.ts

```diff
@@ -191,15 +191,22 @@ export class FlightManifestPlugin {
           ssrNamedModuleId = `./${ssrNamedModuleId.replace(/\\/g, '/')}`
 
         if (isCSSModule) {
+          const chunks = [...chunk.files].filter((f) => f.endsWith('.css'))
           if (!manifest[resource]) {
-            const chunks = [...chunk.files].filter((f) => f.endsWith('.css'))
             manifest[resource] = {
               default: {
                 id,
                 name: 'default',
                 chunks,
               },
             }
+          } else {
+            // It is possible that there are mtuliepl modules with the same resouce,
```

Comment:
```suggestion
            // It is possible that there are multiple modules with the same resource,
```

Reply:
Will address these in my next PR, thanks!